### DISCUSSION TITLE
Consideration for other Operating System

### DISCUSSION THREAD

**Discussion by jumas007** [ag-informatics/ag-informatics-course#49] [Ideas]
Please I would like to suggest that other OS users should be considered as regards the steps for the labwork. Thank you.

**Comment by tame0001** [comment]
Thank you for your suggestion. We try our best to accommodate Mac, Windows, and Linux users. We choose the tools that work across all OSes such as VS code, Git, Python etc. And most instructions are identical. If there are differences, especially installation instructions there are separated sections for each OS. Obviously, we cannot cover all varieties of Linux. Also, the file path will be different, not only because of OS, but also each person preference in managing the file structure. 
If you find any parts that you feel that instructions do not cover your OS of use, please point to those parts. We might miss somethings and will really appreciate your feedback

**Comment by sudokita** [comment]
Thank you all - closing the discussion since OS-specific guides have been updated and provided based on per-cohort needs.
